kv/dist_sender.go

```diff
@@ -304,20 +304,17 @@ func (ds *DistSender) getNodeDescriptor() *roachpb.NodeDescriptor {
 func (ds *DistSender) sendRPC(trace *tracer.Trace, rangeID roachpb.RangeID, replicas replicaSlice, order rpc.OrderingPolicy,
 	ba roachpb.BatchRequest) (*roachpb.BatchResponse, error) {
 	if len(replicas) == 0 {
-		return nil, util.Errorf("replicas set is empty")
+		return nil, noNodeAddrsAvailError{}
 	}
 
 	// Build a slice of replica addresses (if gossiped).
```

Comment:
Ok that makes sense. Non-addressable replicas were the only explanation I could think of for the two checks, but it doesnt look like there is any verification anymore. Do you want me to remove the `" (if gossiped)"` part from this comment?